daemon/src.cmake

```diff
@@ -301,7 +301,7 @@ set(QCOMMONLIST
     ${ENGINE_DIR}/sys/con_common.cpp
 )
 
-if (NOT APPLE)
+if (NOT APPLE AND (USE_CURSES OR WIN32))
```

Comment:
But it wont compile/link if the user overrides it. We could hardcode USE_CURSES on win/apple to ON/OFF while having it configurable on linux. That would also make the checks easier.